packages/gatsby-transformer-sharp/src/customize-schema.js

```diff
@@ -567,6 +568,14 @@ module.exports = ({
   })
 
   if (createTypes) {
-    createTypes([imageSharpType])
+    createTypes([
+      ImageFormatType,
+      ImageFitType,
+      ImageCropFocusType,
+      DuotoneGradientType,
+      PotraceTurnPolicyType,
+      PotraceType,
+      imageSharpType,
```

Comment:
just my nitpick - should it not be like other Types capitalized?

```suggestion
      ImageSharpType,
```

also should changed in this line:

```diff
-  const imageSharpType = schema.buildObjectType({
+  const ImageSharpType = schema.buildObjectType({
```

Reply:
This is not strictly part of this PR, so IMO - let's merge this PR and we can do `style` commit (if anyone feel very strongly about it) separately?